code/datums/components/tippable.dm

```diff
@@ -119,6 +119,9 @@
 		)
 
 		if(!do_after(tipper, tip_time, target = tipped_mob))
+			if(!isnull(tipped_mob.client))
+				tipped_mob.log_message("[key_name(tipper)] failed to tip over [key_name(tipped_mob)]", LOG_ATTACK)
+				tipper.log_message("[key_name(tipper)] failed to tip over [key_name(tipped_mob)]", LOG_ATTACK)
```

Comment:
You might want to take a peek at how other places do attacker/victim logging. Mob logging already adds the mob's name at the start of the log message, and without log_globally=FALSE this log message will be added to log_attack twice.
```
user.log_message("has put [item] on [key_name(source)]", LOG_ATTACK)
source.log_message("had [item] put on them by [key_name(user)]", LOG_VICTIM, log_globally=FALSE)
```

Reply:
Thanks for catching that for me. I did that but forgot to commit it.